Alignment/CommonAlignmentProducer/plugins/BuildFile.xml

```diff
@@ -1,9 +1,10 @@
 <use name="FWCore/MessageLogger"/>
 <use name="FWCore/Framework"/>
 <use name="FWCore/ParameterSet"/>
+<use name="CondFormats/Alignment"/>
```

Comment:
The [build log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ee4b84/40800/build.log) issues a warning:
```
***WARNING: Multiple usage of "CondFormats/Alignment". Please cleanup "use" in "export" section of "src/Alignment/CommonAlignmentProducer/plugins/BuildFile.xml"
```
I think it is because "CondFormats/Alignment" is in "use" both in the common part of this BuildFile, and in the dedicated ones below. (Maybe @makortel can confirm if still around).

If you decide to remove this one, please consider that you removed it at [L48](https://github.com/cms-sw/cmssw/pull/45628/files#diff-5d44c1252caa4ce6fca418edc922ccb02b459e9e143d8a2b564d0d679d8651f8L48), and this should be probably restored. Otherwise, remove [L7](https://github.com/cms-sw/cmssw/pull/45628/files#diff-5d44c1252caa4ce6fca418edc922ccb02b459e9e143d8a2b564d0d679d8651f8R7) and the central one will be used in all compilations.

Reply:
> The [build log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ee4b84/40800/build.log) issues a warning:
> 
> ```
> ***WARNING: Multiple usage of "CondFormats/Alignment". Please cleanup "use" in "export" section of "src/Alignment/CommonAlignmentProducer/plugins/BuildFile.xml"
> ```
> 
> I think it is because "CondFormats/Alignment" is in "use" both in the common part of this BuildFile, and in the dedicated ones below. (Maybe @makortel can confirm if still around).

@perrotta's diagnosis seems correct. @bsunanda please fix (i.e. either use in the common part, if all libraries need it, or only in the specific libraries that need it).

Reply:
Done